lib/livereload.js

```diff
@@ -124,6 +130,13 @@ function StartServer() {
     });
 }
 
+function StopServer() {
+    if (browserSync) {
+        browserSync.exit();
+    }
+    return Q();
```

Comment:
But then it's not really consistent the function itself is synchronous. Either way its not a big deal - just looks confusing.